arpeggio/__init__.py

```diff
@@ -1413,7 +1563,7 @@ class Parser(DebugPrinter):
     FIRST_NOT = Not()
 
     def __init__(self, skipws=True, ws=None, reduce_tree=False, autokwd=False,
-                 ignore_case=False, memoization=False, **kwargs):
+                 ignore_case=False, memoization=False, verbose=False, verbose2=False, **kwargs):
```

Comment:
Why two different verbose flags? What is the purpose of each? Could we just use `verbosity` as a level (int).